code/game/gamemodes/changeling/changeling.dm

```diff
@@ -73,6 +73,12 @@ GLOBAL_VAR(changeling_team_objective_type) //If this is not null, we hand our th
 		log_game("[key_name(changeling)] has been selected as a changeling")
 		var/datum/antagonist/changeling/new_antag = new()
 		new_antag.team_mode = TRUE
+
+		if(changelings.len >= 2)
+			new_antag.competitive_objectives = TRUE
+		else
+			new_antag.competitive_objectives = FALSE
+

```

Comment:
Wouldn't it be better to put new_antag.competetive_objectives = (changelings.len >= 2) instead of this 4 line if else statement?

Reply:
would be cleaner, yeah.